hudi-common/src/main/java/org/apache/hudi/common/model/OverwriteWithLatestAvroPayload.java

```diff
@@ -84,4 +85,12 @@ public Comparable<?> getOrderingValue() {
   public byte[] getRecordBytes() {
     return recordBytes;
   }
+
+  @Override
+  public Option<IndexedRecord> getIndexedRecord(Schema schema, Properties properties) throws IOException {
```

Comment:
should we move this to BaseAvroPayload

Reply:
The interface is defined in `HoodieRecordPayload` and `BaseAvroPayload` does not implement this interface. If we are ok with updating `BaseAvroPayload` to implement the interface then we can move it.